docs/build/qasm-circuits.mdx

```diff
@@ -229,6 +229,6 @@ circuit = loads(program, custom_classical=customs)
 <Admonition type="tip" title="Recommendations">
     - [OpenQASM 3 Qiskit API](/api/qiskit/qasm3)
     - [OpenQASM 2 Qiskit API](/api/qiskit/qasm2)
-    - [Test your program](../test/)
+    - [Test your program](../verify/)
```

Comment:
I won't change this language since we are very soon chopping up this qasm-circuits page